packages/core/src/utils/base64.ts

```diff
@@ -1,3 +1,9 @@
+import { Buffer } from './buffer'
+
 export function base64ToBase64URL(base64: string) {
   return base64.replace(/\+/g, '-').replace(/\//g, '_').replace(/=/g, '')
 }
+
+export function uint8ArrayToBase64URL(array: Uint8Array) {
```

Comment:
We have `TypedArrayEncoded.toBase64Url`. Can we use that instead? That uses `base64ToBase64URL` internally.